src/pages/edge/adobe-journey-optimizer-decisioning/tabs/api-reference.md

```diff
@@ -218,6 +265,91 @@ AEPDecisionScope* decisionScope2 = [[AEPDecisionScope alloc] initWithName: @"myS
 }];
 ```
 
+<Variant platform="ios" api="get-propositions-withTimeout" repeat="12"/>
+
+#### Swift
+
+#### Syntax
+
+```swift
+static func getPropositions(for decisionScopes: [DecisionScope], timeout: TimeInterval,
+                            _ completion: @escaping ([DecisionScope: OptimizeProposition]?, Error?) -> Void)
+```
+
+* _decisionScopes_ is an array of decision scopes for which propositions are requested.
+* _timeout_ is a duration in seconds specifying the maximum time to wait for the event to complete.
+* _completion_ is invoked with propositions dictionary of type `[DecisionScope: OptimizeProposition]`. An `Error` is returned if SDK fails to retrieve the propositions.
```

Comment:
Is `Error` also returned for timeout case?

Reply:
done @spoorthipujariadobe